specs/PermissionManagement.md

```diff
@@ -503,6 +510,12 @@ namespace Microsoft.Web.WebView2.Core
         // developers use the [Local Font Access API](https://wicg.github.io/local-font-access/)
         // to query the system fonts available for styling web content.
         LocalFonts = 10,
+
+        // Indicates permission for a site to use system exlusive messages to access
```

Comment:
"... to send and receive system exclusive messages to/from MIDI devices."

Reply:
Please update